### PR TITLE
Fix/Optimise TopicTrainFragmentTest file.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

I have updated the code to not to launch the same activity twice. These test cases were passing earlier and are still working correctly but this is an optimised code making sure that the activity is not getting launched twice.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is **assigned** to an appropriate reviewer.

### CODE REVIEW

**Comment by rt4914** [comment]
@nikitamarysolomanpvt @veena14cs This PR is ready for review.

**Review by veena14cs — APPROVED** [review]

**Review by veena14cs — APPROVED** [review]
LGTM

File: app/src/sharedTest/java/org/oppia/app/topic/train/TopicTrainFragmentTest.kt [hunk 1]
```diff
@@ -61,7 +61,7 @@ class TopicTrainFragmentTest {
```
Comment (nikitamarysolomanpvt) [inline app/src/sharedTest/java/org/oppia/app/topic/train/TopicTrainFragmentTest.kt:61]: Please remove the empty line 57

**Review by nikitamarysolomanpvt — COMMENTED** [review]
Configuration test cases are failing PTAL

**Pull request closed without merging**